src/content/docs/en/pages/guides/marketplace/templates/bot-basic.mdx

```diff
@@ -0,0 +1,141 @@
+
+---
+
+## Getting the template
+
+To get this template, access [Azion Console](https://console.azion.com/) and click the **+ Create** button on the homepage.
+
+This will open a new page where you can select the **Bot Manager Basic** card.
+
+---
+
+## Setting up the template
+
+On the template's page, click the **Settings** tab to open the configuration form and provide the information to configure your application.
+
+Fields identified with an asterisk are mandatory.
+
+- **Edge Application Name** *: the name of your new edge application on Azion. It'll also be used to name your Edge Firewall instance.
+- **Sample Edge Function Name** *: the name of the sample edge function that will be created.
+Use a unique and easy-to-remember name for these fields.
+
+After completing all the information, click the **Next** button, located in the bottom-right corner. This will start the deployment process.
+
+---
+
+## Deploying the template
+
+You can follow the deployment process through a window showing off the logs. When it's complete, the *deployment* page appears, confirming the edge application has been successfully created.
+
+:::note
+If you didn't launch the Bot Manager function before deploying this template, the execution will fail and a log explaining the reason will be printed.
+:::
+
+This page shows information about the application and some options to continue your journey.
+
+:::note
+The link to the edge application allows you to see it on the browser. However, it takes a certain time to propagate and configure the application in Azion's edge locations. It may be necessary to wait a few minutes for the URL to be activated and for the application page to be effectively displayed in the browser.
+:::
+
+Now you can [manage and adjust the settings](#managing-the-template) through Azin Console.
+
+### Key configurations
+
+This template deploys an entirely new edge configuration including:
+
+- **A new edge application**, with the Edge Functions module enabled, to instantiate your newly created function. The application will receive the requests that trigger the function.
+- **A new domain**, to access your application and test Bot Manager.
+- **A new Edge Firewall instance**, including the instance of the latest version of the Bot Manager integration and a Rules Engine rule to trigger the execution of it.
+- **A new edge function**, providing a basic UI application that can be used to test Bot Manager.
+
+---
+
+## Testing the template
+
+After running the template, you can access the Azion domain, with the format `xxxxxxxxxx.map.azionedge.net/`. It'll show a testing page, where you can send "good" and "bad" requests to the Bot Manager handle.
+
+The testing page will render the response to the request:
+
+- When a *good* request is sent, it'll render a success message.
+- When a *bad* request is sent, it'll render the default `HTTP 403 response status code`, because the Bot Manager's action is set to execute the *deny* action.
```

Comment:
```suggestion
- When a *bad* request is sent, it'll render the default response `HTTP 403 response status code`, because the Bot Manager's action is set to execute the *deny* action.
```